.travis.yml

```diff
@@ -12,15 +12,18 @@ matrix:
         - python: "3.5"
         - python: "3.6"
         - python: "3.7-dev"
-        - env: BUILD_DOCKER=1
+        - env:
+            - BUILD_DOCKER=1
+            - DOCKER_USERNAME=sbrunner
+            - DOCKER_PASSWORD=<replace by `travis encrypt DOCKER_PASSWORD=real-password`>
```

Comment:
To be honest, I'd prefer this to be empty/commented and the script to check if the variables are set instead of it blindly using them. Especially, since we cannot set them in this PR.

Reply:
OK, done, then if there are missing I exit without error :-)